.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/dmlc/dmlc-core
 [submodule "HalideIR"]
 	path = 3rdparty/HalideIR
-	url = https://github.com/dmlc/HalideIR
+	url = https://github.com/jroesch/HalideIR
```

Comment:
use the upstream version

Reply:
There are changes that must be upstreamed, this is so people can build this version and try it out for the time being.